be/src/agent/task_worker_pool.cpp

```diff
@@ -197,6 +197,52 @@ void TaskWorkerPool::submit_task(const TAgentTaskRequest& task) {
     }
 }
 
+void TaskWorkerPool::submit_tasks(std::vector<TAgentTaskRequest>* tasks) {
```

Comment:
use const std::vector<TAgentTaskRequest>& instead.

Reply:
There are modifications here. If you use references, you cannot add const.